--- a/potomo.old	2021-02-26 23:26:16.325386100 +0200
+++ a/potomo	2021-02-26 23:26:24.545386100 +0200
@@ -16,7 +16,7 @@
 # gettext can only cope with utf-8 encoded mo files; thus we make this
 # sure while creating the mo.  A conversion is not done if the source
 # file does not exist or if it is not newer than the mo file.
-
+exit 0
 if [ "$1" = "--get-linguas" -a $# -eq 2 ]; then
    if [ ! -f "$2/LINGUAS" ]; then
        echo "potomo: directory '$2' has no LINGUAS file" >&2
